src/main/webapp/WEB-INF/jsp/roster.jsp

```diff
@@ -117,9 +117,9 @@
                                     });
                                 });
                             </script>
-                            <div class="col-md-3 saveAttendanceButton">
-                                <input id="saveAttendanceOnTop" class="hovering-purple-button" type="submit" name="saveAttendance"
-                                       value="Save Attendance"/>
+                            <div class="col-md-4 saveAttendanceButton">
+                                <button id="saveAttendanceOnTop" class="hovering-purple-button" type="button" name="saveAttendance" onclick="submitRoster()">
+                                    Save Attendance</button>
                             </div>
                             <div class="col-md-4 deleteAttendanceButton">
                                 <a id="deleteAttendance" href="${context}/roster/${selectedSectionId}/delete" name="deleteAttendance" style="text-decoration: none" >
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this deprecated "name" attribute. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AS1827)